azurerm/internal/services/network/data_source_virtual_network_gateway_connection.go

```diff
@@ -109,6 +109,29 @@ func dataSourceArmVirtualNetworkGatewayConnection() *schema.Resource {
 				Computed: true,
 			},
 
+			"traffic_selector_policy": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
```

Comment:
can there be more then 1 here? or should this me maxitems of 1

Reply:
Yep, you're right. I'll add `MaxItems: 1`